src/main/js/main_html5.js

```diff
@@ -1371,7 +1392,9 @@ import CONSTANTS from "./constants/constants";
      * @param {object} event The event from the video
      */
     var raiseDurationChange = _.bind(function(event) {
-      raisePlayhead(this.controller.EVENTS.DURATION_CHANGE, event);
+      if (!isLive || isDvrAvailable()) {
```

Comment:
Are there any side-effects to omitting this notification for Live streams that don't have DVR?

Reply:
@jpfranco sorry I forgot to revert this. Was trying to avoid another minor issue with the playhead flashing to the start of the dvr window if failover happens on the live edge. I think this may introduce other consequences though as you suggest